build_vm_compilers test failure on windows

Found as part of #2999, and we have been unable to reproduce manually.  Test disabled until after the dependency graph untangling in progress.

```
Run pub.bat run test
00:00 +0: test\vm_kernel_integration_test.dart: can compile vm apps to kernel (setUpAll)
00:00 +0 -1: test\vm_kernel_integration_test.dart: can compile vm apps to kernel (setUpAll) [E]
  Expected: <0>
    Actual: <69>
  Because every version of build_modules from path depends on scratch_space any which is forbidden, build_modules from path is forbidden.
  So, because a depends on build_modules from path, version solving failed.
  
  
  package:test_api                    expect
  package:_test_common/sdk.dart 22:3  pubGet
  
00:00 +0 -1: test\vm_kernel_integration_test.dart: can compile vm apps to kernel (tearDownAll)
00:00 +0 -1: Some tests failed.
Error: Process completed with exit code 1.
```

--- a/build_vm_compilers/test/vm_kernel_integration_test.dart
+++ b/build_vm_compilers/test/vm_kernel_integration_test.dart
@@ -1,4 +1,3 @@
-@OnPlatform({'windows': Skip('https://github.com/dart-lang/build/issues/3001')})
 // Copyright (c) 2018, the Dart project authors.  Please see the AUTHORS file
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.

